themes-default/slim/views/vue-components/file-browser.mako

```diff
@@ -138,8 +140,8 @@ Vue.component('file-browser', {
     },
     mounted() {
         const vm = this;
-
         let fileBrowserDialog;
+        this.currentPath = this.initialDir;
```

Comment:
This didnt work here. Thats whay i added. I dont know why the merhod was not triggered for this page

Reply:
Which page is that?

Reply:
SearchResults

Reply:
Oh I think I know why!
I added that watch to make sure if initialDir is updated AFTER the component is created (if you get the value from an API call) it would update the current value.
In this case the value isn't updated because it's available directly from Python.

**TL;DR:** You're correct in adding this line, but I think changing `currentPath` in `data()` to `this.initialDir` is better :) (On line 84)